elyra/pipeline/processor_kfp.py

```diff
@@ -329,8 +329,6 @@ def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwri
             description = f'Created with Elyra {__version__} pipeline editor using {pipeline.source}.'
 
             for key, operation in defined_pipeline.items():
-                if operation.classifier not in ["execute-notebook-node", "execute-python-node", "execute-r-node"]:
-                    continue
                 self.log.debug("component:\n "
```

Comment:
Since this loop exists solely to dump debug output, it might be a good use for `self.log.isEnabledFor(logging.DEBUG)` to wrap the `for` loop.  I think we should also add a precursor statement to the effect of "Exporting pipeline '{pipeline_name}' with components..." to give some context to all of the components getting logged.

I would also recommend using f-strings to reduce code surface within the loop.